zuora.js

```diff
@@ -111,10 +111,10 @@ Zuora.prototype.authenticate = function() {
 };
 
 Zuora.prototype.getObject = function(url) {
-  var self = this;
+  const self = this;
```

Comment:
Just to be picky, I think this variable assignment is useless as we use an arrow function below, so `this` is still the same :slightly_smiling_face: